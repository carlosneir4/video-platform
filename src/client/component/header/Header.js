import React, { Fragment } from 'react';
import { Link } from 'react-router-dom';


const Header = () => {

  return (
    <nav>
      <div className="Header">
        <Link to="/" className="left brand-logo">
          Video Platform
            </Link>
        <ul className="right">
          <li>
            <Link to="/new">New Video</Link>
          </li>
          <li>
            <Link to="/list">List Videos</Link>
          </li>
        </ul>
      </div>
    </nav>
  );
};

export default Header;