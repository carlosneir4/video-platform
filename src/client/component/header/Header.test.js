import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'
import Header from './Header'

configure({
  adapter: new Adapter()
})

describe('Testing Component <Header />', () => {
    test('should Snapshot met', () => {

        const wrapper = shallow(<Header />)
        expect(toJson(wrapper)).toMatchSnapshot()
      });
});