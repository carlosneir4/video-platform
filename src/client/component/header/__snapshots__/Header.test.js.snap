// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Component <Header /> should Snapshot met 1`] = `
<nav>
  <div
    className="Header"
  >
    <Link
      className="left brand-logo"
      to="/"
    >
      Video Platform
    </Link>
    <ul
      className="right"
    >
      <li>
        <Link
          to="/new"
        >
          New Video
        </Link>
      </li>
      <li>
        <Link
          to="/list"
        >
          List Videos
        </Link>
      </li>
    </ul>
  </div>
</nav>
`;
